loader/loader.c

```diff
@@ -42,8 +43,13 @@
 // DON'T REMOVE is for reducing binary size.
 // These funtios are defined as weak in /libc/src/init.c
 //--------------------------------------------------------------
-void _ps2sdk_libc_init() {}
-void _ps2sdk_libc_deinit() {}
+void _libcglue_init() {}
+void _libcglue_deinit() {}
+void _libcglue_args_parse(int argc, char **argv) {}
```

Comment:
if we override ags parse we have to setup the CWD path manually. this broke PS2BBL relative paths back then